crates/holochain/src/core/workflow/sys_validation_workflow/tests.rs

```diff
@@ -131,14 +131,15 @@ async fn sys_validation_produces_forked_chain_warrant() {
     let record = records.unwrap();
     let (action, _) = record.into_inner();
     let mut action = action.into_inner().0.into_content();
-    let entry = Entry::App(::fixt::fixt!(AppEntryBytes));
+    let entry = Entry::App(AppEntryBytes(UnsafeBytes::from(vec![11; 11]).into()));
```

Comment:
What's the reason to not use fixt here? Was it making the test flaky or was it just unnecessary?

Reply:
Yes, this made! the test flaky, because this fixt will use `()` as the data with high probability, which is the same as the other data being created, leading to no fork. Good to remember that fixturators are arbitrary but not random.